gnd/src/test/java/com/google/android/gnd/ui/editobservation/DateFieldViewModelTest.java

```diff
@@ -61,14 +61,14 @@ public void setup() {
   @Test
   public void testUpdateResponse() {
     dateFieldViewModel.updateResponse(DATE);
-    Observer<Optional<Response>> optionalObserver = new Observer<Optional<Response>>() {
+    // Observer initialized to read the value of live data.
+    dateFieldViewModel.getResponse().observeForever(new Observer<Optional<Response>>() {
```

Comment:
Still a bit confused; it looks like all this does it remove the observer on change, but it doesn't affect anything else. What am I missing?